compass/landice/tests/greenland/mesh.py

```diff
@@ -64,9 +76,112 @@ def run(self):
             gridded_dataset='greenland_1km_2024_01_29.epsg3413.icesheetonly.nc',  # noqa
             projection='gis-gimp', geojson_file=None)
 
+        # Create scrip files if they don't already exist
+        if exists(data_path + '/BedMachineGreenland-v5.scrip.nc'):
+            logger.info('BedMachine script file exists;',
+                        ' skipping file creation')
```

Comment:
The logger expects a single argument (I think), so this line causes an error in the log file (but doesn't make the test fail). I'm just removing the comma, since you don't need it for string continuation and that suppresses the logging error 

```suggestion
            logger.info('BedMachine script file exists;'
                        ' skipping file creation')
```